src/features/commands/premium.js

```diff
@@ -0,0 +1,74 @@
+const { SlashCommandBuilder } = require('@discordjs/builders')
+const { Permissions } = require('discord.js');
+const db = require('../../db');
+
+module.exports = {
+    type: 'slash',
+    data: new SlashCommandBuilder()
+        .setName("premium")
+        .setDescription('Configure premium servers')
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName("configure")
+                .setDescription("Configure premium features.")
+                .addRoleOption(option => 
+                    option
+                        .setName("role")
+                        .setDescription("The role that will be assigned to toxic members.")
+                        .setRequired(true))
+                .addChannelOption(option =>
+                    option
+                        .setName("channel")
+                        .setDescription("The investigations channel.")
+                        .setRequired(true)))
+        .addSubcommand(subcommand =>
+            subcommand
+                .setName('disable')
+                .setDescription('Disables all premium features.')),
+    async execute(_, interaction) {
+        if (!interaction.guild) {
+            return interaction.reply({ content: 'You can only use this command in servers!', ephemeral: true });
+        }
+
+        if (!interaction.member.permissions.has(Permissions.FLAGS.ADMINISTRATOR)) {
+            const ohSimusAsset = new MessageAttachment('./src/assets/images/ohsimus.png');
+            return interaction.reply({
+                content: 'Oh snap! You don\'t have sufficient permissions to execute this command.',
+                files: [ohSimusAsset],
+                ephemeral: true
+            });
+        }
+
+        const isPremium = await db.getServerConfig(interaction.guild.id).then(config => config[0]?.isSubscribed)
+        if (!isPremium) {
+            return interaction.reply({
+                content: 'This command can only be run on a premium server, learn more over at [Dotsimus.com](https://dotsimus.com/).',
```

Comment:
Markdown links don't work in message content, it only works in embeds

```suggestion
                content: 'This command can only be run on a premium server, learn more over at https://dotsimus.com.',
```

Reply:
I just copied that from report.js, it might be apparent there as well.

Reply:
The link markdown works in report works tho.
![image](https://user-images.githubusercontent.com/29563098/141686334-3b6ddd6c-f34d-4e51-be11-0decb989bd2e.png)

Reply:
Have you tested this? Looks like you didn't

Reply:
![image](https://user-images.githubusercontent.com/29563098/141704735-a2151cfe-0b16-480c-b0a9-07882371db0b.png)
It seems that link markdown is exclusive to bots.

Reply:
I've did some testing and found that it works in interaction responses, but not in normal messages.

Reply:
I'll mark this as resolved for now since it doesn't look like a problem.